devdocs/orientation-guide.md

```diff
@@ -0,0 +1,131 @@
+<center>![](images/teammates_heading.png) </center>
+
+
+# **Contributor Orientation Guide**
+
+
+
+We welcome contributions from anyone, in particular, students (see [here](https://www.google.com/url?q=http://teammatesv4.appspot.com/about.html&sa=D&ust=1458427634357000&usg=AFQjCNGIbS5nfk0HxVCMSqPXXmcaOSz47w) for a list of our contributors). One of the main objectives of TEAMMATES is to help students get experience in a OSS production environment. Here are some information that might be useful to would-be contributors.
+
+
+
+[Knowledge required](#h.abftcg92xk1k)
+
+[Workload](#h.8yn20iebwilp)
+
+[Getting started](#h.4es3v3yquxal)
+
+[Contacting us](#h.5lngkuxtl5mo)
+
+* * *
+
+
+
+## Knowledge required
+
+Although TEAMMATES uses many tools and technologies, you need not know all of them before you can contribute. The diagram below shows which technologies you need to learn to contribute in different roles. As you can see, some roles don't need learning any technologies at all. Note that these role aren't fixed or formally assigned. It is simply for guidance only.
+
+<center>
+
+![RolesAndTechnologies.png](https://lh6.googleusercontent.com/3cvr4dEJEBiqlsY_X6-sTwTyYCvk_d5YKsLM2Z-IWI7AEd6c1bYgZ4LDW5yDbwrd9b3kIrGD08kIKyVkg9vHmO-_iXhs8bpGQyJnwpKA12pFWj1Tj-XYuXVYkOBgoI_BVBHkR6I)
+
+</center>
+
+Roles:
+
+* **Tester**: Tests the App manually and reports bugs or suggestions for enhancements in the issue tracker.
+*   **Copy editor**: Helps in improving documentation.
+*   **Web page developer**: Works on static web pages, such as those used in the TEAMMATES web site
+*   **Test developer**: Works on automating system tests.
+*   **Frontend developer**: Works on the frontend of the App that is generated dynamically from the server side.
+*   **Backend developer**: Works on the backend logic of the App, including data storage.
+
+## Workload
```

Comment:
@damithc could we consider changing this title? Maybe something like "Team Structure" or "Member Contribution Levels" or "Team Hierarchy" ...?